website/docs/r/spring_cloud_app.html.markdown

```diff
@@ -78,7 +78,11 @@ An `custom_persistent_disk` block exports the following:
 
 An `identity` block supports the following:
 
-* `type` - (Required) Specifies the type of Managed Service Identity that should be configured on this Cosmos Account. The only possible value is `SystemAssigned`.
+* `type` - (Required) Specifies the type of Managed Service Identity that should be configured on this Spring Cloud Application. Possible values are `SystemAssigned`, `UserAssigned`, `SystemAssigned, UserAssigned` (to enable both).
```

Comment:
According to Managed Identity wiki, there is no space for `SystemAssigned,UserAssigned`, I'm wondering if we should keep consistent with them.

See [Managed Identity wiki](https://armwiki.azurewebsites.net/authorization/managed_identities/MSIOnboardingUserAssigned.html).

Reply:
Terraform provides uniformed UX on identity block, and it suggests use `SystemAssigned, UserAssigned`, but it can handle it if backend returns `SystemAssigned,UserAssigned`

Reply:
Great to know this!